swagger.json

```diff
@@ -531,6 +531,325 @@
+                }
+              }
+            }
+          },
+          "400": {
+            "description": "The sign-off content does not meet requirements",
+            "content": {
+              "application/json": {
+                "schema": {
+                  "$ref": "#/components/schemas/Errors"
+                }
+              }
+            }
+          },
+          "401": {
+            "description": "The submission is missing a valid API token in header",
+            "content": {
+              "application/json": {
+                "schema": {
+                  "$ref": "#/components/schemas/Errors"
+                }
+              }
+            }
+          }
+        }
+      }
+    },
+    "/project/{projectIdentifier}": {
+      "get": {
+        "summary": "Get a project archive record",
+        "description": "This resource represents a project and archive record. Each project and archive record is identified by a string `projectIdentifier`",
+        "parameters": [
+          {
+            "name": "projectIdentifier",
+            "in": "path",
+            "description": "unique project identifier",
+            "required": true,
+            "schema": {
+              "type": "string"
+            }
+          },
+          {
+            "name": "includeVidarrMetadata",
+            "in": "query",
+            "description": "Whether or not to return the full Vidarr metadata for files sent offsite and files sent to vidarr-archival (true/false)",
+            "required": false,
+            "schema": {
+              "type": "boolean"
+            }
+          }
+        ],
+        "responses": {
+          "201": {
+            "description": "Created project archive record",
+            "content": {
+              "application/json": {
+                "schema": {
+                  "$ref": "#/components/schemas/Project-response"
+                }
+              }
+            }
+          },
+          "404": {
+            "description": "The requested resource was not found"
+          }
+        }
+      }
+    },
+    "/project/{projectIdentifier}/copied-to-offsite-staging": {
+      "put": {
+        "summary": "Project archive files moved to offsite staging directory",
+        "description": "Indicate that this project archive's files have been moved to the offfsite staging directory",
+        "parameters": [
+          {
+            "name": "projectIdentifier",
+            "in": "path",
+            "description": "unique project identifier",
+            "required": true,
+            "schema": {
+              "type": "string"
+            }
+          }
+        ],
+        "requestBody": {
+          "required": true,
+          "description": "Vidarr copy-out file and archiving batch ID",
+          "content": {
+            "application/json": {
+              "schema": "#/components/schemas/ProjectArchive-copied-to-offsite-staging-dir"
```

Comment:
There's a pre-existing error in this endpoint for case archives, would you be able to fix both? Instead of 
```
"schema": "#/components/..."
```
it should be 
```
"schema": {
  "$ref": "#/components/..."
}
```